tests/lib/Gateway/DoctrineSchemaMetadataRegistryTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\CorePersistence\Gateway;
+
+use Ibexa\CorePersistence\Gateway\DoctrineSchemaMetadataRegistry;
+use LogicException;
+use PHPUnit\Framework\TestCase;
+
+final class DoctrineSchemaMetadataRegistryTest extends TestCase
```

Comment:
This test seems to be incomplete

Reply:
Technically it was simply moved from PC, where we did not have tests for this at all. Some unit + integration tests will eventually be necessary to ensure stability of course, but at this moment I'm mostly concerned about actually separating it from that library :sweat_smile: